packages/teams-js/src/public/meeting.ts

```diff
@@ -925,6 +926,47 @@ export namespace meeting {
     );
   }
 
+  /**
+   * This function is used to join a meeting
+   * @param joinMeetingParams
+   */
+  export function joinMeeting(joinMeetingParams: JoinMeetingParams): Promise<string | object> {
+    return new Promise<string | object>((resolve) => {
+      ensureInitialized(runtime, FrameContexts.content);
+      resolve(
+        sendAndHandleStatusAndReasonWithVersion(
+          getApiVersionTag(ApiVersionNumber.V_2, ApiName.Meeting_JoinMeeting),
+          'meeting.joinMeeting',
+          joinMeetingParams,
+        ),
+      );
+    });
+  }
+
+  /** Join meeting parameters */
+  export interface JoinMeetingParams {
+    /** The meeting url to join. */
+    skypeTeamsMeetingUrl: string;
+    /** The subject of the meeting. */
+    subject: string;
+    /** The conversation id of the meeting. */
+    source: EventActionSource;
+    /** The conversation id of the meeting. */
+    conversationId: string;
```

Comment:
Are all of the various ids here actually just strings, or would they be more correct to be Guids / UUIDs? Even if they have to be just strings, I assume you will want to do *some* validation in the function above to guarantee that they are non-null, non-empty, etc.

Reply:
If the ids should be uuid, see `marketplaceUtils.validateUuid` for an example on how to make sure the UUID is a legal format.